docs/counter/index.md

```diff
@@ -46,36 +45,47 @@ to decorate `make` with the
 let's just say that `@react.component` is there to reduce boilerplate and make
 our code more readable and easier to maintain.
 
+## Wrap strings with `React.string`
+
+Unlike in normal React, we must wrap strings inside function calls to convert
+them to the `React.element` type. This is exactly what the `React.string`
+function does---if you hover over it, you'll see that it displays the type
+`string => React.element`.
+
+## Using the App component
 
 A little bit further down, we make use of the `App` component:
 
-<<< ./Snippets.re#use-app-component
+<<< Snippets.re#use-app-component
```

Comment:
Why was the `./` dropped? Just not necessary?

Reply:
Yeah, when I first added it I assumed it was needed, but it's not